app/drivers/sensor/CMakeLists.txt

```
@@ -2,4 +2,5 @@
 # SPDX-License-Identifier: MIT
 
 add_subdirectory_ifdef(CONFIG_ZMK_BATTERY battery)
-add_subdirectory_ifdef(CONFIG_EC11 ec11)
\ No newline at end of file
+add_subdirectory_ifdef(CONFIG_EC11 ec11)
+add_subdirectory_ifdef(CONFIG_PMW33xx pmw33xx)
```

I think there's a small typo here
```suggestion
add_subdirectory_ifdef(CONFIG_PMW33XX pmw33xx)
```